floyd/cli/run.py

```diff
@@ -194,26 +198,29 @@ def run(ctx, gpu, env, message, data, mode, open, tensorboard, gpup, cpup, comma
         floyd_logger.info("   floyd logs %s", job_name)
 
 
-def get_command_line(gpu, env, message, data, mode, open, tensorboard, command):
+def get_command_line(instance_type, env, message, data, mode, open, tensorboard, command_str):
     """
     Return a string representing the full floyd command entered in the command line
     """
-    floyd_command = "floyd run"
+    floyd_command = ["floyd", "run"]
+    floyd_command.append('--' + INSTANCE_NAME_MAP[instance_type])
     if not env == DEFAULT_ENV:
-        floyd_command = floyd_command + " --env {}".format(env)
+        floyd_command += ["--env", env]
     if message:
-        floyd_command = floyd_command + " --message \"{}\"".format(message)
+        floyd_command += ["--message", shell_quote(message)]
     if data:
         for data_item in data:
-            floyd_command = floyd_command + " --data {}".format(data_item)
-    if not mode == "job":
-        floyd_command = floyd_command + " --mode {}".format(mode)
-    if gpu:
-        floyd_command = floyd_command + " --gpu"
-    if not open:
-        floyd_command = floyd_command + " --no-open"
+            floyd_command += ["--data", data_item]
     if tensorboard:
-        floyd_command = floyd_command + " --tensorboard"
-    if command:
-        floyd_command = floyd_command + " \"{}\"".format(' '.join(command))
-    return floyd_command
+        floyd_command.append("--tensorboard")
+    if not mode == "job":
+        floyd_command += ["--mode", mode]
+        if mode == 'jupyter':
+            if open:
+                floyd_command.append("--open")
```

Comment:
This is the default, we probably don't have to append this since most users will not specify this.